### ISSUE TITLE
❌ ♿ Accordion elements are not properly announced for nonsighted users

### ISSUE THREAD

**Issue by dgullerud-usc** [elementor/elementor#36170]
### Description

- The accordion widget is not marked up properly. In the accordion widget, aria-selected="true" is being used on the button. This is an invalid use of aria-selected. For an accordion, aria-expanded and aria-controls should be used instead. 

- When using the VoiceOver screen reader, the buttons are not announced correctly. 

- The buttons cannot be activated with space bar as a proper button normally would. 

### Steps to reproduce

Go to https://lgbtqplus.usc.edu/involvement/student-employment/
With VoiceOver, JAWS or NVDA activated, navigate to the Accordion buttons "Programming assistant, Operations assistant, Outreach assistant" by pressing tab. What is announced by VoiceOver is "link Programming assistant, link Operations assistant, link Outreach assistant." This is not announced correctly.

Next, with focus on any of the buttons (e.g. Programming assistant) try activating the button by pressing space bar. The panel does not expand or collapse. 

### Expected behavior

Here is an example of an accessible accordion that is marked up correctly. 
https://dequeuniversity.com/library/aria/accordion-single

VoiceOver announces each accordion panel / button as "[label], collapsed / expanded, button". This element has the attribute aria-exanded="true/false" and aria-controls="sect1". Valid and proper aria attributes on the button. Also, pressing spacebar on the accordion button expands and collapses the panel since this is a button, not a link. 

### Elementor System Info

```txt
Production environment
Elementor Version 4.1.1 
Theme: Neve
```

### Agreement

- [ ] I confirm I have read and followed all the guidelines and instructions outlined in the Elementor Bug Report form.
- [x] I agree that my issue may be closed without further action if it doesn't meet all the requirements outlined in the Elementor Bug Report form.

**Comment by nicholaszein** [comment]
Hello. 

Your issue lack crucial information required both in the [Contribution Guidelines for Reporting Bugs](https://elemn.to/gh-contributing-bug-reports), and [Bug Report Form](https://elemn.to/gh-new-bug-report), making it impossible to investigate the problem. 

The full Elementor System Info and steps to reproduce are essential for a successful investigation. 

You are also using the legacy Accordion widget. This issue was solved with the Nested Accordion. Please enable Containers on your Elementor > Editor > Settings > Features page, and replace your current legacy Accordion widget with the Nested Accordion widget for a more accessible output.

For this reason I'm closing this issue.

Regards
